app/assets/stylesheets/_books.scss

```diff
@@ -1,22 +1,51 @@
+$border-color: #E8E8E8;
+
+.filters{
+  @include span-columns(2);
+  .genres li {
+    @include padding(15px null 15px null);
+    border-bottom: 1px solid $border-color;
+  }
+}
+
+.books-list {
+  @include span-columns(10);
+}
+
 .book {
   @include outer-container;
   @include fill-parent();
 }
 
+
+.book-details > div {
+  @include padding(15px null 15px null);
+  border-bottom: 1px solid $border-color;
+}
+
 .book-category {
   width: $base-book-width;
 }
 
-.book__title {
+.book-title {
   color: $base-secondary-color;
   font-size: $base-title-size;
 }
 
-.book__genre {
+.book-rating {
+  font-weight: bold;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size, font-weight, text-align, width